src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/SelectionManager.cs

```diff
@@ -8,25 +8,25 @@
 namespace System.Windows.Forms.Design.Behavior;
 
 /// <summary>
-///  The SelectionBehavior is pushed onto the BehaviorStack in response to a positively hit tested SelectionGlyph.  The SelectionBehavior performs  two main tasks: 1) forward messages to the related ControlDesigner, and 2) calls upon the SelectionManager to push a potential DragBehavior.
+///  The SelectionBehavior is pushed onto the BehaviorStack in response to a positively hit tested SelectionGlyph. The SelectionBehavior performs  two main tasks: 1) forward messages to the related ControlDesigner, and 2) calls upon the SelectionManager to push a potential DragBehavior.
 /// </summary>
 internal sealed class SelectionManager : IDisposable
 {
     private BehaviorService _behaviorService;           // ptr back to our BehaviorService
     private IServiceProvider _serviceProvider;          // standard service provider
     private readonly Dictionary<IComponent, ControlDesigner> _componentToDesigner;    // used for quick look up of designers related to components
     private readonly Control _rootComponent;            // root component being designed
-    private ISelectionService _selectionService;                  // we cache the selection service for perf.
-    private IDesignerHost _designerHost;                // we cache the designerhost for perf.
-    private Rectangle[]? _previousSelectionBounds;           // used to only repaint the changing part of the selection
-    private object? _previousPrimarySelection;               // used to check if the primary selection changed
+    private ISelectionService _selectionService;        // we cache the selection service for perf.
+    private IDesignerHost _designerHost;                // we cache the designerHost for perf.
```

Comment:
```suggestion
    private IDesignerHost _designerHost;                // we cache the DesignerHost for perf.
```
alternatively we can remove most of these comments and replace the useful ones(lines 21,22,25) with comments on the line above the field definition. If you choose to re-do comments please do it in a new PR.

Reply:
I will undo change and move up on a new PR. that just fixes spelling issues. Moving comment up is probably a separate issue.